bibliothecarius-service/src/main/java/com/peony/bibliothecarius/service/application/service/AuthorService.java

```diff
@@ -0,0 +1,45 @@
+package com.peony.bibliothecarius.service.application.service;
+
+import com.peony.bibliothecarius.api.AuthorDTO;
+import com.peony.bibliothecarius.service.domain.entity.Author;
+import com.peony.bibliothecarius.service.domain.repository.AuthorRepository;
+import org.modelmapper.ModelMapper;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@Service
+public class AuthorService {
+
+    private final AuthorRepository authorRepository;
+    private final ModelMapper modelMapper = new ModelMapper();
+
+    public AuthorService(AuthorRepository authorRepository) {
+        this.authorRepository = authorRepository;
+
+        modelMapper.typeMap(Author.class, AuthorDTO.class)
+                .addMappings(mapper -> mapper.skip((author, authorDTO) -> author.setId(null)));
+    }
+
+    public Author create(AuthorDTO authorDTO) {
+        return authorRepository.save(modelMapper.map(authorDTO, Author.class));
+    }
+
+    public List<Author> getAll() {
+        return authorRepository.findAll();
+    }
+
+    public Author getById(String id) {
+        return authorRepository.findById(id).orElseThrow(() -> new RuntimeException("Author not found by id=" + id));
+    }
+
+    public Author update(String id, AuthorDTO authorDTO) {
+        Author authorFromDb = getById(id);
+        modelMapper.map(authorDTO, authorFromDb);
+        return authorRepository.save(authorFromDb);
```

Comment:
```suggestion
        return authorRepository.save(
               modelMapper.map(authorDTO, getById(id))
        );
```

prefer not to declare variable if it is used only once, unless it significantly improves readability